deploy/operator.yaml

```diff
@@ -2735,7 +2754,7 @@ spec:
               name: "metrics"
               protocol: "TCP"
           # Replace this with the built image name
-          image: quay.io/kubevirt/hostpath-provisioner-operator:latest
+          image: registry:5000/hostpath-provisioner-operator:latest
```

Comment:
This means we'll see the issue in https://github.com/kubevirt/hostpath-provisioner-operator/pull/169 again?

Reply:
yeah, I need to make a check to ensure I am not introducing that in a PR. This definitely needs to get fixed.